docs/development.md

```diff
@@ -19,14 +19,14 @@ branch, but release branches should not change.
 
 ## Prerequisites
 
-OpenPitrix only has few external dependencies you need to setup before being 
+OpenPitrix only has few external dependencies you need to setup before being
 able to build and run the code.
 
 ### Setting up Go
 
 OpenPitrix written in the [Go](http://golang.org) programming language.
-To build, you'll need a Go (version 1.8+) development environment. 
-If you haven't set up a Go development environment, please follow 
+To build, you'll need a Go (version 1.8+) development environment.
```

Comment:
Go 1.9+

Reply:
Go must be 1.9+?

Reply:
make all will invoke go test, some test helper func requires Go 1.9. (t.Helper)

Reply:
https://golang.org/doc/go1.9#test-helper